aten/src/ATen/core/NamedTensor.h

```diff
@@ -91,7 +91,7 @@ struct TORCH_API NoNamesGuard {
       reset();
     }
   }
-  void reset() {
+  void reset() const {
```

Comment:
Resetting it is inherently non-const, right?

Reply:
It's semantically non-const, but in this case the object is not modified.